pkg/instancetype/preference/find/revision.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors
+ *
+ */
+package find
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/cache"
+
+	virtv1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/instancetype/find"
+)
+
+type RevisionFinder struct {
+	controllerRevisionFinder *find.ControllerRevisionFinder
+}
+
+func NewRevisionFinder(store cache.Store, virtClient kubecli.KubevirtClient) *RevisionFinder {
```

Comment:
Same question as for the instancetype RevisionFinder. Does this provide any additional benefits over using ControllerRevisionFinder?

Reply:
Just centralising the `NamespacedName` creation really but it's worth it IMHO.